lib/bloc/backup/backup_bloc.dart

```diff
@@ -517,7 +517,7 @@ class BackupBloc {
       }
 
       if (request.encryptionKey != null && request.encryptionKey.key != null) {
-        assert(request.encryptionKey.key.length > 0 || true);
+        assert(request.encryptionKey.key.isNotEmpty || true);
```

Comment:
I think this assertion will never trigger if you have an `|| true`

Reply:
Yes, it is true stupid legacy code from the Breez.